core/src/test/java/org/infinispan/stream/BaseStreamTest.java

```diff
@@ -219,7 +218,7 @@ public void testObjSortedCollector() {
 
       List<Map.Entry<Integer, String>> list = createStream(entrySet).sorted(
             (e1, e2) -> Integer.compare(e1.getKey(), e2.getKey())).collect(
-            CacheCollectors.serializableCollector(Collectors::toList));
+            () -> Collectors.toList());
```

Comment:
Man I wish that nested generics work a bit better.
Sad that the `serializableCollector` works but not the new way :(

Reply:
Yeah :(. I'm going to try to build a small test case that's easier to share with StackOverflow or submit as bug to Oracle.

Reply:
@galderz So you have decided not to use wildcard for accumulator type?

Reply:
I've been able to replicate [this standalone](https://gist.github.com/galderz/a44468018e5ecd270fd7e0e28db5b499) and I've submitted a [question](https://stackoverflow.com/questions/46274846/java-compilation-fail-with-method-reference-but-works-with-lambda) to StackOverflow. Let's see if we can get an answer.

Reply:
Fix for this line of code: https://stackoverflow.com/a/46276537/186429

Reply:
I didn't know you could force types with method references. FYI of others:

    Collectors::<Map.Entry<Integer, String>>toList